feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/HomeMapbox.kt

```diff
@@ -146,24 +148,31 @@ private fun MapboxMap(
     // so we have to make sure the map is loaded before setting location
     val onStyleLoadedListener = OnStyleLoadedListener { event ->
         event.end?.run {
-            when (state.feedingRouteState) {
+            when (state.feedingPointState.feedingRouteState) {
                 is FeedingRouteState.Disabled -> mapboxMapView.setLocation(state.locationState.location)
                 is FeedingRouteState.Active -> setLocationOnRoute(mapboxMapView, state)
             }
         }
     }
 
-    LaunchedEffect(key1 = state.feedingPoints) {
-        markerController.drawMarkers(
-            feedingPoints = state.feedingPoints
-        )
+    LaunchedEffect(key1 = state.feedingPointState.feedingPoints) {
+        if (state.feedingPointState.feedingRouteState is FeedingRouteState.Disabled) {
```

Comment:
And this one

Reply:
I removed background to fix issue when switching animal types